Makefile

```diff
@@ -121,13 +116,6 @@ docker-test-integration:
 		-v $(CURDIR):/usr/src/github.com/vmware-tanzu/antrea:ro \
 		antrea/test test-integration $(USERID) $(GRPID)
 
-.PHONY: docker-tidy
-docker-tidy: $(DOCKER_CACHE)
```

Comment:
I thought the purpose of `docker-tidy` was to ensure anyone could generate the proper go.mod and go.sum files regardless of the version of go they are using locally?

Reply:
In this way, I think we can keep docker-tidy. Will update this part.

Reply:
Done.